### ISSUE TITLE
feature request: option to send certificate for NGINX ssl client verification

### ISSUE THREAD

**Issue by renehonig** [gotify/android#85]
I am using Gotify in combination with NGINX (based on the example configuration script). I would like to use the [NGINX ssl client verification option](https://nginx.org/en/docs/http/ngx_http_ssl_module.html#ssl_client_certificate) (syntax: `ssl_client_certificate file; ssl_verify_client on`), which would require the Gotify app to send a pre-configured client certificate. Could this feature be added, or is there another/better way to do this?

**Comment by jmattheis** [comment]
Yeah, this could be done.

**Comment by MoonCactus** [comment]
I suspect my problem is identical
* in use the docker version of gotify server (Gotify version 2.0.21@2021-01-06-15:38:32), latest from yesterday
* it is set up as a local HTTP service
* I use nginx as a proxy (enventually with the provided config example as well, my early tests were no better)
* I use an external https://push.mydomain.com URL with a letsencrypt certificate
* The web UI works well (when I publish with curl or gotify CLI I do get the notification in my browser and desktop popup)
* The prebuilt android client does log in correctly to the server

Though I can see client's calls to the docker server like `DELETE  "/message"`, the android client never shows any notifcation.

I am not sure how to investigate further (eg. which logfiles in the docker, or how to check if the websocket is working?)

**Comment by jmattheis** [comment]
@MoonCactus have you tried sending a message with a higher priority? See https://github.com/gotify/android#message-priorities
If this doesn't work please create a new issue.

**Comment by MoonCactus** [comment]
Thanks, yes! `curl 'https://push.yaknou.com/message?token=my_token" -F "title=from_pc" -F "message=msg from $(hostname)" -F "priority=9"`

**Comment by MoonCactus** [comment]
(actually, the more I think about it the more I think I am polluting the initial report anyway, sorry!)

**Comment by MoonCactus** [comment]
PEBCAC? Today I deleted all my users all the app tokens, then created them again from scratch -- and it worked immediately.
 
I also was not able to reproduce the problem! Just in case you wanted to know or it happended again: I was properly logged with the same user on the desktop and on the smartphone. When I posted a notification with curl, I was notified on the desktop, but never on the smartphone. I checked that the websockets were established on the server, and the log in the smartphone app showed no error either. It may be that I deleted/re-created the tokens more than once.
 
Note that this probably **does not close the original FR** though -- I was offtopic, sorry again for that!
